docs/user-guide/workflow.ipynb

```diff
@@ -164,9 +167,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "he3_workflow = pol.he3.He3CellWorkflow(in_situ=False)\n",
+    "he3_polarizer_workflow = pol.He3CellWorkflow(in_situ=False, incoming_polarized=False)\n",
+    "he3_analyzer_workflow = pol.He3CellWorkflow(in_situ=False, incoming_polarized=True)\n",
+    "sm_workflow = pol.supermirror.SupermirrorWorkflow()\n",
```

Comment:
Unused.

Reply:
Same as above. its for the overview for the reader to see (i) all possibilities and then (ii) an example. Hence, I will add another text to describe that.